integration-tests/src/test/java/com/datastax/dse/driver/api/core/graph/statement/GraphTraversalITBase.java

```diff
@@ -595,10 +595,8 @@ public void should_allow_use_of_dsl_graph_binary() throws Exception {
    */
   @Test
   public void should_return_correct_results_when_bulked() {
-    Assumptions.assumeThat(
-            ccmRule().getCcmBridge().getDseVersion().get().compareTo(Version.parse("5.1.2")) > 0)
-        .isTrue();
-
+    assumeThat(ccmRule().getDseVersion())
+        .hasValueSatisfying(version -> assumeThat(version).isGreaterThan(Version.parse("5.1.2")));
```

Comment:
I had to change this in order to migrate the return type of `ccmRule()`: child classes now return `CcmRule`, which doesn't have a `getCcmBridge()` method.
And `getDseVersion()` is a simpler way to do it anyway.